src/rdiffbackup/locations/_repo_shadow.py

```diff
@@ -812,11 +812,10 @@ def get_increment_times(cls, rp=None):
         return return_list
 
     @classmethod
-    def _initialize_restore(cls, data_dir, restore_to_time):
+    def _initialize_restore(cls, restore_to_time):
```

Comment:
That's all ? I'm not sure what is the impact of the change.

Reply:
None, that was the point, it was just removing a useless action.